code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -127,6 +128,22 @@
 	can_be_sawn_off = FALSE
 	can_jam = FALSE
 
+/obj/item/gun/ballistic/rifle/boltaction/brand_new/prime
+	name = "Regal Nagant"
+	desc = "A prized hunting Mosin Nagant. Used for the most dangerous game."
+	icon_state = "moistprime"
+	inhand_icon_state = "moistprime"
+	worn_icon_state = "moistprime"
+	can_be_sawn_off = TRUE
+	sawn_desc = "An sawn-off Regal Nagant... Doing this was a sin, I hope you're happy. \
+		You are now probably one of the few people in the universe to ever hold a \"Regal Obrez\". \
+		Even thinking about that name combination makes you ill."
+
+/obj/item/gun/ballistic/rifle/boltaction/brand_new/prime/sawoff(mob/user)
+	. = ..()
+	if(.)
+		name = "Regal Obrez" // wear it loud and proud
```

Comment:
```suggestion
		name = "\improper Regal Obrez" // wear it loud and proud
```
Also, does the parent not call it an obrez after being sawn off? That's surprising.

Reply:
yeah a mosin's name becomes "sawn-off Mosin Nagant"